frontend/src/redux/drawerModeSlice.ts

```diff
@@ -0,0 +1,35 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { KubeObject } from '../lib/k8s/KubeObject';
+
+interface DrawerModeState {
+  isDetailDrawerEnabled: boolean;
+  selectedResource: KubeObject | undefined;
+}
+
+const getLocalDrawerStatus = (key: string) => {
+  const local = localStorage.getItem(key);
```

Comment:
can be replaced with `(key: string) => localStorage.getItem(key) === 'true' `

Reply:
done!